packages/contracts-bedrock/contracts/test/OptimismPortal.t.sol

```diff
@@ -318,43 +505,33 @@ contract OptimismPortal_Test is Portal_Initializer {
         uint8 _gasLimit,
         bytes memory _data
     ) external {
-        // Cannot call the optimism portal
-        vm.assume(_target != address(op));
```

Comment:
Why did you remove this `vm.assume`? It will eventually find an invalid input without it

Reply:
My mistake.